README.md

```diff
@@ -79,6 +79,13 @@ Run the test suite:
 npx ts-node demo/test-suite.ts
 ```
 
+Run the test on a particular test case (requires the identifier of the test case):
+
+```sh
+npx ts-node mode/test-case.ts <test-case-id>
```

Comment:
There appears to be a small type here. 
Should be **demo**/test-case.ts instead of `npx ts-node mode/test-case.ts`